pyproject.toml

```diff
@@ -903,7 +903,8 @@ module = [
     "tests.sentry.sentry_metrics.limiters.test_writes_limiter",
     "tests.sentry.sentry_metrics.test_base_indexer",
     "tests.sentry.sentry_metrics.test_batch",
-    "tests.sentry.sentry_metrics.test_multiprocess_steps",
+    "tests.sentry.sentry_metrics.test_gen_metrics_multiprocess_steps",
+    "tests.sentry.sentry_metrics.test_rh_metrics_multiprocess_steps",
     "tests.sentry.snuba.metrics.test_snql",
```

Comment:
new files should not be excluded from type checking

Reply:
I see, we're actually splitting an old file into 2 here, they're basically same code. Do we still need to add typing to it?

Reply:
yes

Reply:
Remove the file names from `pyproject.toml`, marking this as resolved